docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md

```diff
@@ -75,6 +75,64 @@ all incoming calls to the target broker, but it would also be possible to stop
 the message from interception by other interceptors and even to block it from
 dispatch to the broker.
 
+### Implementing a tenant-providing interceptor
+
+:::note
+Camunda 8 relies on Identity for tenant management. Tenant-providing interceptors are only compatible with Zeebe, and
```

Comment:
I would link to https://docs.camunda.io/docs/next/self-managed/identity/user-guide/tenants/managing-tenants/ here.